docs/SUMMARY.md

```diff
@@ -2,6 +2,9 @@
 
 - [Catalogue](README.md)
 - [Search](search/README.md)
+- [Hypotheses](search/hypotheses/README.md)
+- [Analysis](search/analysis/README.md)
+- [Query design](search/query-design/README.md)
```

Comment:
We've now made these sections (intentions / relevance tests etc) subsections of "query design". I'm not sure that's what we want?

Assuming we want these new sections as subsections of search? If so they need a double spacing indent. 

Something like

```suggestion
  - [Hypotheses](search/hypotheses/README.md)
  - [Analysis](search/analysis/README.md)
  - [Query design](search/query-design/README.md)
```

Reply:
You can see the changes in markdown format by clicking this button above the document:

![viewchange](https://user-images.githubusercontent.com/31692/90116489-772ed680-dd4d-11ea-902d-29ed69d99fb9.png)

Reply:
I only want the three sections as subsections of Search, not Query design.

Reply:
Yip. The markdown format to make them subsections is to indent them with two spaces below as the above example shows.

Reply:
Ok, I'll try to resolve this myself.

Reply:
Bearing in mind this is my first time making any explicit coding changes in Github, I clicked on Resolve conflicts and deleted the headings under Query design and indented the remaining 3 headings so that they appear under Search (as below) but Mark as resolved remains greyed.
![image](https://user-images.githubusercontent.com/32418070/90118855-bf032d00-dd50-11ea-8768-bfff3055cc99.png)

Reply:
The new sections are now visible in GitBook but apparently the lambda test failed.  I'm all in here. Over to you guys.